demos/calibration/calibration-pubmed-node-classification.ipynb

```diff
@@ -0,0 +1,3065 @@
+{
```

Comment:
<p>May be it makes sense to rouynd 4 digits after comma as then we could see that the results are slightly different</p>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/326/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='115'/>